packages/frontend/src/components/Claim/WinBid/WinBidForm.tsx

```diff
@@ -38,22 +40,25 @@ export const WinBidForm = ({
   setVoucher,
 }: WinBidFormProps) => {
   const luck = win !== undefined
+  const { claimingEndTime } = useClaimingEndTime()
+  const FEE = 2
+  const withdrawalAmount = useMemo(() => bid.sub(bid.mul(FEE).div(100)), [bid])
```

Comment:
We should pass `withdrawalAmount` to next view (ReviewForm), because it's still bidAmount.